src/app/components/LiveLabel/index.tsx

```diff
@@ -8,22 +8,42 @@ import styles from './index.styles';
 import Text from '../Text';
 
 interface LiveLabelProps {
-  ariaHidden?: boolean;
-  liveText?: string;
   offScreenText?: string;
   lang?: string;
   id?: string;
 }
 
 const LiveLabel = ({
-  ariaHidden = false,
-  liveText = 'LIVE',
-  offScreenText = '',
   lang = 'en-GB',
   id = '',
   children,
+  offScreenText,
```

Comment:
Reminder that id should not be initialised - not sure if we managed to pull in these changes from the other branch?

```suggestion
  id,
  children,
  offScreenText,
```

Reply:
this change has now been pulled in